projects/azure/RELEASES.md

```diff
@@ -0,0 +1,22 @@
+# OSA Release tracker
```

Comment:
we need something like this, but I'm not sure this is the place.  Leaving it in to see if it sticks.